onmt/trainer.py

```diff
@@ -496,14 +496,15 @@ def _gradient_accumulation(
                         bptt = True
 
                         # 3. Compute loss.
+                        # The loss of the prompt will be set to zero.
+                        batch = self.train_loss.ignore_prompt(batch)
```

Comment:
We might want to enable/disable this with some dedicated opt flag. (It should be transparent in theory, but probably not useful to make it run when not needed.)